uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/token/TokenMvcMockTests.java

```diff
@@ -722,10 +724,6 @@ public void testOpenIdToken() throws Exception {
             .session(session)
             .param(OAuth2Utils.GRANT_TYPE, "authorization_code")
             .param("code", code)
-            .param(OAuth2Utils.RESPONSE_TYPE, "token id_token")
-            .param(OAuth2Utils.SCOPE, "openid")
```

Comment:
The scope parameter MUST be present, and contain openid as a scope. Otherwise the behavior is unmodified